feeds.conf

```diff
@@ -1,5 +1,5 @@
-src-git packages git://github.com/openwrt/packages.git^f43f01ea157c24704c75c376dffd15008a6006bd
-src-git luci git://github.com/openwrt/luci.git^70a4d43cc895b7d728b4fc201f2b6fd9f4b8aaec
+src-git packages git://github.com/openwrt/packages.git^e245e2b8246d0d25892d3dc2af0d3834ef95a02e
+src-git luci git://github.com/openwrt/luci.git^6f5c46681b55790bfcc2d81c3933b1d434a5400f
 src-git routing git://github.com/openwrt-routing/packages.git^d580d71309f0867022ab7f47badb8c2a1860fc83
```

Comment:
Ja, und zwar gibt es kein "gutes" Ziel für eine Aktualisierung. In for-15.05 ist https://github.com/openwrt-routing/packages/commit/f7ffb388b17fd327e85f7e0c386a9d175b14e44e (immer) noch nicht drin, und master passt naturgemäß (eigentlich) nicht zu CC. Ich mache mal einen PR.